### PR TITLE
Add extensible UploadNoStyle

### PR DESCRIPTION
**Pull request by wraeth-eth**
## What does this PR do and why?

This attempts to remove as much styling choices from Upload to allow full styling and customization, while leveraging the antd class/es. I wonder if it might just be easier to rebuild however…

## Screenshots or screen recordings

_If applicable, provide screenshots or screen recordings to demonstrate the changes._

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by wraeth-eth** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2156** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2156" target="_blank">Graphite</a> 
    * **PR #2169** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2169" target="_blank">Graphite</a> 
      * **PR #2170** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2170" target="_blank">Graphite</a> 
        * **PR #2171** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2171" target="_blank">Graphite</a> 
          * **PR #2172** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2172" target="_blank">Graphite</a> 
            * **PR #2181** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2181" target="_blank">Graphite</a> 
              * **PR #2182** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2182" target="_blank">Graphite</a> 
                * **PR #2183** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2183" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2183?utm_source=stack-comment).

File: src/components/UploadNoStyle/UploadNoStyle.tsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    undo: VoidFunction
+  }) => ReactNode
+}
+
+export const UploadNoStyle = (props: UploadNoStyleProps) => {
+  const [_uploadUrl, _setUploadUrl] = useState<string>()
+  const uploadUrl = props.value ?? _uploadUrl
+  const setUploadUrl = props.onChange ?? _setUploadUrl
+
+  const [isUploading, setIsUploading] = useState<boolean>(false)
```
Comment (tomquirk) [inline src/components/UploadNoStyle/UploadNoStyle.tsx:29]: **nitpick** just as an org thing, i tend to like grouping `useStates` together. NBD. I wonder if we could lint for it.

**Review by tomquirk — APPROVED** [review]

File: src/components/UploadNoStyle/UploadNoStyle.tsx (hunk 1 above)
Comment (wraeth-eth) [reply src/components/UploadNoStyle/UploadNoStyle.tsx:29]: yeah me too - will fix

**Comment by wraeth-eth** [comment]
**Graphite Merge Job**

Current status: ⏳ Queued to merge

This pull request is currently queued to be merged as part of a stack.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2183?utm_source=stack-comment).


Job Reference: vtVBnYDD1aIW70DZJ1cT

**Pull request merged**